app/main.py

```diff
@@ -2,4 +2,73 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: float,
+            coords: list[int] = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        if coords is None:
+            self.coords = [0, 0]
+        else:
+            self.coords = coords
+
+    def go_forward(self, step: int = 1) -> None:
+        self.coords[1] += step
+
+    def go_back(self, step: int = 1) -> None:
+        self.coords[1] -= step
+
+    def go_right(self, step: int = 1) -> None:
+        self.coords[0] += step
+
+    def go_left(self, step: int = 1) -> None:
+        self.coords[0] -= step
+
+    def get_info(self) -> str:
+        print(f"Robot: {self.name}, "
+              f"Weight: {self.weight},")
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(
+            self,
+            name: str,
+            weight: float,
+            coords: list[int] = None
+    ) -> None:
+        super().__init__(name, weight, coords)
+        self.coords = coords or [0, 0, 0]
```

Comment:
There is no need to redefine `self.coords`, you could pass a necessary value straight away to the `super().__init__()` 
```suggestion
        super().__init__(name, weight, coords or [0, 0, 0])
```